packages/@webex/plugin-meetings/test/unit/spec/meeting/muteState.js

```diff
@@ -17,11 +17,17 @@ describe('plugin-meetings', () => {
 
   const createFakeLocalStream = (id, muted) => {
     return {
-      id,
+      setServerMuted: sinon.spy(),
+      setUnmuteAllowed: sinon.spy(),
       setMuted: sinon.stub(),
-      setServerMuted: sinon.stub(),
-      setUnmuteAllowed: sinon.stub(),
-      muted,
+      outputStream: {
```

Comment:
I'm not sure if this is correct. MuteState reads the muted property of the stream in the MuteState.init() method and webrtc-core implementation for "muted" getter reads the "enabled" property of the inputTrack while you're setting it as the "muted" property of the outputTrack, so both the object and property seem to be wrong. This affects the setup of the tests, so once this is fixed maybe you will not need some of the other true->false and false->true changes that you did in the tests

Reply:
I agree. Turns out the muteState reads the `muted` of the fakeLocalTrack created which is not actual webrtc-core object. So, the getter wasn't invoked. 

Removing the outputStream felt right. But do we want to make this createFakeLocalTrack to return an actual LocalTrack?

Reply:
do you need an actual LocalTrack for anything? I don't mind it much, so do what is easier. Usually with a fake one it's easier to control how it behaves, but you can use a real one and just stub specific methods on it too. If you use a real one then the name "createFake..." should probably be changed, because it won't be fake anymore

Reply:
I agree. We actually don't need an actual LocalTrack. So let this be like this. We will revisit in future if required